diff --git a/python/onos/config/admin/__init__.py b/python/onos/config/admin/__init__.py
index 24868f2..b79742b 100644
--- a/python/onos/config/admin/__init__.py
+++ b/python/onos/config/admin/__init__.py
@@ -8,7 +8,7 @@ from typing import AsyncIterable, AsyncIterator, Dict, Iterable, List, Union
 import betterproto
 from betterproto.grpc.grpclib_server import ServiceBase
 import grpclib
-
+import gnmi.proto
 
 class Type(betterproto.Enum):
     """Streaming event type"""
@@ -96,7 +96,7 @@ class ModelInfo(betterproto.Message):
     # model_data is a set of metadata about the YANG files that went in to
     # generating the model plugin. It includes name, version and organization for
     # each YANG file, similar to how they are represented in gNMI Capabilities.
-    model_data: List["___gnmi__.ModelData"] = betterproto.message_field(3)
+    model_data: List["gnmi.proto.ModelData"] = betterproto.message_field(3)
     # module is the name of the Model Plugin on the file system - usually ending
     # in .so.<version>.
     module: str = betterproto.string_field(4)
@@ -409,6 +409,5 @@ class ConfigAdminServiceBase(ServiceBase):
         }
 
 
-from .... import gnmi as ___gnmi__
 from ..change import device as _change_device__
 from ..snapshot import device as _snapshot_device__
